src/05-derivatives/01-introduction.md

```diff
@@ -173,6 +177,27 @@ sub-01/func/sub-01_task-rest_space-individual_bold.json
 }
 ```
 
+Preprocessed `bold` CIFTI-2 files that have been sampled to the fsLR surface
+meshes defined in the Conte69 atlas along with the MNI152NLin6Asym template.
+In this example, because all volumetric structures are sampled to the same
+reference, the `VolumeReference` key is used as a default, and only the
+surface references need to be specified by BrainStructure names.
+
+```Text
+sub-01/func/sub-01_task-rest_space-fsLR_den-91k_bold.dtseries.nii
+sub-01/func/sub-01_task-rest_space-fsLR_den-91k_bold.json
+```
+
+```JSON
+{
+    "SpatialReference": {
+        "VolumeReference": "https://templateflow.s3.amazonaws.com/tpl-MNI152NLin6Asym_res-02_T1w.nii.gz",
```

Comment:
should we stick with constant names from CIFTI-2 for volume as well?
```suggestion
        "VOLUME": "https://templateflow.s3.amazonaws.com/tpl-MNI152NLin6Asym_res-02_T1w.nii.gz",
```

Reply:
i'm ok with changing to `VOLUME` but @oesteban where in cifti-2 did you find that constant?

Reply:
just in case it carries other semantics

Reply:
Perhaps I can explain my reasoning:

1) A CIFTI-2 file, as currently generated by fMRIPrep, has 22 BrainModels. `CORTEX_LEFT`, `CORTEX_RIGHT` and a bunch of volumetric ones.
2) There is a `CIFTI_STRUCTURE_ALL_GREY_MATTER` and `CIFTI_STRUCTURE_OTHER_GREY_MATTER`.
   1. These are not in an ontology, where it can be deduced that, when looking for the reference for some grey matter structure, one should also check this entry.
   2. These are hypothetically structures that could be directly encoded, which would lead to an ambiguity of whether the reference was meant specifically for those structures or, again, all grey matter.
3) In the absence of a relevant CIFTI-2 constant, we are imposing something in BIDS. The BIDS convention for JSON keys is CamelCase.
4) In the event that this becomes relevant to another file format, it is not guaranteed that we will get constants that in CAPITAL_SNAKE.

I'm not deeply opposed to VOLUME, but unlike the others, using doesn't reduce the impedance mismatch for CIFTI-2. So I'd lean towards BIDS-like conventions.